projects/core/src/occ/occ-models/occ.models.ts

```diff
@@ -1469,64 +1469,6 @@ export namespace Occ {
      * @member {boolean} [updateable]
      */
     updateable?: boolean;
-    /**
-     * @member {StatusSummary[]} [statusSummaryList]
-     */
-    statusSummaryList?: StatusSummary[];
-    /**
-     * @member {ConfigurationInfo[]} [configurationInfos]
-     */
-    configurationInfos?: ConfigurationInfo[];
-  }
-
-  /**
-   *
-   * An interface representing ConfigurationInfo.
-   * Provides information about configuration values of the entry.
-   */
-  export interface ConfigurationInfo {
-    /**
-     * @member {string} [configurationLabel]
-     */
-    configurationLabel?: string;
-    /**
-     * @member {string} [configurationValue]
-     */
-    configurationValue?: string;
-    /**
-     * @member {string} [configuratorType]
-     */
-    configuratorType?: string;
-    /**
-     * @member {string} [status]
-     */
-    status?: string;
-  }
-
-  /**
-   * Possible order entry statuses
-   */
-  export enum OrderEntryStatus {
-    Success = 'SUCCESS',
-    Info = 'INFO',
-    Warning = 'WARNING',
-    Error = 'ERROR',
-  }
-
-  /**
-   *
-   * An interface representing StatusSummary.
-   * Provides status including number of issues for configurable entry.
-   */
-  export interface StatusSummary {
-    /**
-     * @member {number} [numberOfIssues]
-     */
-    numberOfIssues?: number;
-    /**
-     * @member {string} [status]
-     */
-    status?: OrderEntryStatus;
```

Comment:
Until we have a solution for splitting the `Occ` namespace in between the libraries, let's keep all the Occ models in this namespace in the core library.
(we've done the same in for the MyCompany feature).